account_invoice_company_search/__openerp__.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 ADHOC SA (<http://adhoc.com.ar>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Account Invioce Company Search",
+    "version": "1.0",
+    'author':  'ADHOC',
+    'website': 'www.adhoc.com.ar',
+    "category": "Accounting",
+    "description": """
+Account Invioce Company Search
+==============================
+Adds company search filter in account invoice view
```

Comment:
To be well displayed on apps.odoo.com, starting from the version 8.0, we remove the description key  from `__openerp__.py` and instead we add a README.rst  file based on this template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst